pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -45,12 +68,33 @@ func (oe openshiftExtension) Transformers(comp v1alpha1.TektonComponent) []mf.Tr
 		occommon.ApplyCABundles,
 	}
 }
+
 func (oe openshiftExtension) PreReconcile(ctx context.Context, tc v1alpha1.TektonComponent) error {
 	return nil
 }
-func (oe openshiftExtension) PostReconcile(context.Context, v1alpha1.TektonComponent) error {
-	return nil
+func (oe openshiftExtension) PostReconcile(ctx context.Context, tc v1alpha1.TektonComponent) error {
+	return oe.installerSetClient.PostSet(ctx, tc, oe.routeManifest, filterAndTransform())
 }
-func (oe openshiftExtension) Finalize(context.Context, v1alpha1.TektonComponent) error {
-	return nil
+
+func (oe openshiftExtension) Finalize(ctx context.Context, tc v1alpha1.TektonComponent) error {
+	err := oe.installerSetClient.CleanupPostSet(ctx)
+	return err
+}
+
+func getRouteManifest() (*mf.Manifest, error) {
+	manifest := &mf.Manifest{}
+	resultsRbac := filepath.Join(common.ComponentBaseDir(), routeRBACYamlDirectory)
+	if err := common.AppendManifest(manifest, resultsRbac); err != nil {
+		return nil, err
+	}
+	return manifest, nil
+}
+
+func filterAndTransform() client.FilterAndTransform {
```

Comment:
can we move this to transformer?

Reply:
It's not reusable because other packages also use some other transformers. Here we are already leveraging `common.Transform`.